packages/manager/modules/new-billing/src/autoRenew/autorenew-translation.service.js

```diff
@@ -0,0 +1,15 @@
+export default class {
+  /* @ngInject */
+  constructor($translate, coreConfig) {
+    this.$translate = $translate;
+    this.coreConfig = coreConfig;
+  }
+
+  descriptionOfHeading() {
+    const region = this.coreConfig.getRegion();
+    const description = this.$translate.instant('billing_description');
+    const details = this.$translate.instant('billing_description_details');
+
+    return region !== 'US' ? `${description} ${details}` : description;
```

Comment:
This could be improved to avoid unnecessary computation:
```suggestion
    const region = this.coreConfig.getRegion();
    const description = this.$translate.instant('billing_description');
    if (region === 'US') return description;
    
    const details = this.$translate.instant('billing_description_details');

    return `${description} ${details}`;
```

_This change is not mandatory, since the method is called only once_